snuba/cli/devserver.py

```diff
@@ -135,6 +135,16 @@ def devserver(*, bootstrap: bool, workers: bool) -> None:
                 "--consumer-group=cdc_group",
             ],
         ),
+        (
+            "profiles",
+            [
+                "snuba",
+                "consumer",
+                "--auto-offset-reset=latest",
+                "--log-level=debug",
+                "--storage=profiles",
+            ],
+        ),
```

Comment:
Do you want to turn this on by default for Sentry dev already? The migrations are still skipped so it won't really work unless that switch is flipped too. May be better to keep this line behind the ENABLE_DEV_FEATURES flag or something.